robottelo/hosts.py

```diff
@@ -1134,8 +1134,8 @@ def virt_who_hypervisor_config(
         ]
         repos.extend(extra_repos)
         content_setup_data = cli_factory.setup_cdn_and_custom_repos_content(
-            org['id'],
-            lce['id'],
+            org_id,
+            lce_id,
```

Comment:
Reading above these lines, it looks like there is logic to create an org or lce if `org_id` or `lce_id` is passed in as `None`. Is there a reason you changed this?

Reply:
Honestly, I made this change in February and have no idea why I did. I'll switch it back and retest, then update here if there's a good reason for this change. Otherwise, I'll revert it.